src/contrib/ethosu/cascader/plan_generator.cc

```diff
@@ -105,7 +105,8 @@ std::vector<bool> GetCascadableAxes(const Part& part) {
   return cascadable_axes;
 }
 
-std::vector<StripeConfig> GenerateOutputStripeConfigs(const Part& part, int stripe_factors) {
+std::vector<StripeConfig> GenerateOutputStripeConfigs(const Part& part, int stripe_factors,
+                                                      bool disable_striping = false) {
```

Comment:
Nit: probably no need for a default here?

Reply:
Agreed and removed the default.